### PR TITLE
ADAPT-1913 Created List Style field options for Card Variants

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Created List Style field options for Card Variants

# Review By (Date)
- 05/05

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Clear cache
3. Create a page with Cards variants.
[See screenshots within ticket for how Simple and Bordered list styles should appear.](https://stanfordits.atlassian.net/browse/ADAPT-1913)
4. Review page styles
Note: The Horizontal Card Bordered List Styles will not span full-width.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ADAPT-1913](https://stanfordits.atlassian.net/browse/ADAPT-1913)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: saa_victoria_subtheme.react_behaviors.yml [hunk 1]
```diff
@@ -23,6 +23,14 @@ card_variant_info:  # part of the key after the :
       options:
         su-card-wrapper--border-none: Minimal (no border)
         su-card-wrapper--border-regular: Border and Drop Shadow
+    card_list_style:
+      type: select
+      title: 'Card List Style'
+      description: 'Select the style for list format. First paragraph tag will appear regular and all following paragraphs will be styled as a list.'
+      options:
+        su-card-wrapper--no-style: No List Style
```
Comment (rebeccahongsf) [inline saa_victoria_subtheme.react_behaviors.yml:31]: Katria had made a note to create 2 boolean fields within [this ticket](https://stanfordits.atlassian.net/browse/ADAPT-1914). I believe that we would only want the editor to be able to choose one of the two styles so I decided to create a select list instead. 🤔 

I created the `su-card-wrapper--no-style : No List Style` as the default list style in the event that the editor would like to unselect the list style.

**Comment by jenbreese** [comment]
The bordered list style is a little off center vertically. That's the only visual thing I've found so far.

File: saa_victoria_subtheme.react_behaviors.yml (hunk 1 above)
Comment (jenbreese) [reply saa_victoria_subtheme.react_behaviors.yml:31]: Oh good. I like the no list style for the default.

**Review by jenbreese — APPROVED** [review]
Good to go. Just that vertically centered in the bordered list.

**Comment by jenbreese** [comment]
Thanks! That is vertically centered now on my local test pages.

**Pull request merged**